recipe-app/View/CustomCollectionViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomCollectionViewCell.swift
+//  recipe-app
+//
+//  Created by Arthur Hermann on 29/11/2024.
+//
+
+import UIKit
+
+final class CustomCollectionViewCell: UICollectionViewCell {
+    
+    static let identifier = "CustomCollectionViewCell"
```

Comment:
We don't need this dependency. You are using it on one place so its fine to use a sting.